cluster/stone-prd-rh01/tenants/crt-redhat-acm-tenant/maestro/overlay/maestro/main/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+nameSuffix: -main # Add a suffix to all resource names in the application for uniqueness
```

Comment:
This will apply to everything in the namespace but the optics will be workspace-wide, do we not expect to need to version this at all?

Reply:
The next version of this component will actually be a component in ACM so imo this is desirable!